scripts/portfolio.js

```diff
@@ -0,0 +1,24 @@
+var projects = [];
+
+function Project (repos) {
+  // constructor function for github repos to be displayed
+  this.title = repos.title;
+  this.updatedOn = repos.updatedOn;
+  this.body = repos.body;
+}
+
+Project.prototype.toHtml = function () {
+  var $newProject = $('article.template').clone();
+
+  $newProject.attr('data-title', this.title);
+  $newProject.attr('data-updatedOn', this.updatedOn);
+  $newProject.attr('data-body', this.body);
```

Comment:
You don't need to be adding the entire body of your project as a data attribute.